xbmc/Application.cpp

```diff
@@ -1081,10 +1081,30 @@ bool CApplication::InitDirectoriesLinux()
   else
     userHome = "/root";
 
+  /* Get XDG paths.
+   * Though a slash is expected at the end anyway, let's cover ourselves, just in case. */
+  std::string xdgDataPath;
+  if (getenv("XDG_DATA_HOME"))
+  {
+    xdgDataPath = getenv("XDG_DATA_HOME");
+    URIUtils::AddSlashAtEnd(xdgDataPath);
```

Comment:
I'm keen on giving adding that a go. Would an envvar for this be the preferable approach, or something else entirely?